scripts/tools/metric.yml

```diff
@@ -165,6 +165,7 @@ class: $tMetricGroup
 name: CalculateMultipleMetricValues
 decl: static
 details:
+    - "Application must call firstly pSetCount and pMetricValueCount to know the size to allocate for pMetricCountPerSet and pMetricValues, respectively."
```

Comment:
`"Application must query values of pSetCount and pMetricValueCount to know the size to allocate for pMetricCountPerSet and pMetricValues, respectively."`